unabridged-works-volume-1/original/14-boston.adoc

```diff
@@ -46,9 +46,10 @@ Even so Father; for so it seemed good in thy sight.
 Matt. 11:25-26
 ____
 
-=== TO THE RULERS, TEACHERS, AND PEOPLE OF New England
+[.blurb]
+=== To the Ruler New England
```

Comment:
Looks like we accidentally lost some of the words of this heading.

Reply:
thanks for catching that, I probably got interrupted mid-sentence. 😆